app/views/claims/_check_your_answers_section.html.erb

```diff
@@ -1,21 +1,11 @@
-<h2 class="govuk-heading-m">
-  <%= heading %>
-</h2>
+<% unless heading.nil? %>
+  <h2 class="govuk-heading-m">
+    <%= heading %>
+  </h2>
+<% end %>
 
 <dl class="govuk-summary-list govuk-!-margin-bottom-9">
   <%- answers.each do |(label, answer, slug)| %>
-    <div class="govuk-summary-list__row">
-      <dt class="govuk-summary-list__key">
-        <%= label %>
-      </dt>
-
-      <dd class="govuk-summary-list__value">
-        <%= answer %>
-      </dd>
-
-      <dd class="govuk-summary-list__actions">
-        <%= link_to('Change', claim_path(slug), class: 'govuk-link') unless slug.nil? %>
-      </dd>
-    </div>
+    <%= render partial: "answer_row", locals: { label: label, answer: answer, slug: slug } %>
```

Comment:
I don't think we gain anything for having this partial anymore. Can it get rolled back in to this one?